File: Python/2014/Vote Count.py
# J2

numvotes = int(input())
votes = input()
acount = 0
bcount = 0

for i in votes:
    if i == 'A':
        acount += 1
    elif i == 'B':
        bcount += 1

if acount > bcount:
    print('A')
elif bcount > acount:
    print('B')
else:
    print('Tie')

File: Java/2014/VoteCount.java
// J2

import java.util.Scanner;

public class VoteCount {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int numvotes = scanner.nextInt(), a_count = 0, b_count = 0;

        String votes = scanner.next();

        for (int i = 0; i < numvotes; i++) {
            if (votes.charAt(i) == 'A') a_count++;
            else b_count++;
        }

        scanner.close();

        if (a_count > b_count) System.out.println('A');
        else if (b_count > a_count) System.out.println('B');
        else System.out.println("Tie");
    }
}